src/exercises/Exercises.js

```diff
@@ -121,16 +135,20 @@ export default function Exercises({
   }
 
   function defineExerciseSession(batches, rest, bookmark_count) {
+    let exerciseList = EXERCISE_TYPES
```

Comment:
exerciseList sounds to me like a list of exercises. But what we have is actually a list of exercise types. so maybe rename to `exerciseTypesList`? it's longer, but more precise I think.